src/engines/synths/nuke/nuke.cpp

```diff
@@ -88,6 +87,8 @@ namespace otto::engines {
     return true;
   }
 
+#pragma GCC diagnostic push 
+#pragma GCC diagnostic ignored "-Wreturn-type"
```

Comment:
I have no idea why this is the warning to disable for this, but lets roll with it